### PR TITLE
Give farming cave creeps the same scaling as basic camps

### PR DESCRIPTION
**Pull request by chrisinajar**
this might also shine light on how bad the creep scaling is....

### CODE REVIEW

File: game/resource/English/modifier/items/modifier_charge_replenishing.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+    //=============================================================================
+    // Generic Charge Replenisher
+    //=============================================================================
+            "DOTA_Tooltip_modifier_charge_replenishing"                          "Replenishing charges"
```
Comment (Chronophylos) [inline game/resource/English/modifier/items/modifier_charge_replenishing.txt]: Is this suppose to be in here?

**Comment by Chronophylos** [comment]
I thought Haganeko wanted Gold and XP so high. But also Creeps to be stronger.

File: game/resource/English/modifier/items/modifier_charge_replenishing.txt (hunk 1 above)
Comment (chrisinajar) [inline game/resource/English/modifier/items/modifier_charge_replenishing.txt]: No, it was supposed to be in my other PR and i fucked it up and then just left it

**Comment by chrisinajar** [comment]
yeah, but it can't be worse than what's in there now. They probably still need to be stronger and give less XP.

**Pull request merged**